src/javascripts/ng-admin/es6/lib/View/View.js

```diff
@@ -195,10 +191,10 @@ class View {
     }
 
     processFieldsDefaultValue(entry) {
-        for (var i in this._fields) {
-            var field = this._fields[i];
+
+        this._fields.forEach(function (field) {
             entry.values[field.name()] = field.defaultValue();
-        }
+        });
 
         return this;
```

Comment:
Not your fault, but this method should return the modified `entry`, not the current view.